src/common.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+//Panics if `err` is not `nil`
+//
+// @param err Error to be checked for equality to `nil`
+func panicIfError(err error) {
```

Comment:
May be we should write some log here? And is it good error-handling pattern? (I really dont know. But i know a guy, who knows, i'll ask him)

Reply:
I agree with logging. But I don't know about good patterns of error-handling patterns, it's just an idea a came up with, and this function was one of the reasons to request your review